packages/apollo/src/commands/client/codegen.ts

```diff
@@ -64,6 +64,9 @@ export default class Generate extends ClientCommand {
     namespace: flags.string({
       description: "The namespace to emit generated code into."
     }),
+    omitDeprecatedEnumCases: flags.boolean({
+      description: "Omit deprecated enum cases from generated code [Swift only]"
```

Comment:
Should this default to `false` to preserve existing behavior, or would that be overkill?

Reply:
Yeah I intended for this to default to false to preserve existing behavior- is that not how it would behave as currently implemented? Looking at the implementation of `flags.boolean` I saw "specifying a default of false is the same not specifying a default"

Reply:
Ah ok (not exactly a typescript expert over here). I think that should work fine, then.